source/Calamari/Kubernetes/ResourceStatus/ResourceUpdateReporter.cs

```diff
@@ -82,24 +83,24 @@ private void SendServiceMessage(Resource resource, bool removed, int checkCount)
             
             var parameters = new Dictionary<string, string>
```

Comment:
Some of these parameters are inferred from the deployment context now - which hasn't been updated here.

If you're up for it, should be a quick fix
https://github.com/OctopusDeploy/OctopusDeploy/blob/main/source/Octopus.Core/Features/Kubernetes/ServiceMessages/Observability/KubernetesDeploymentStatusServiceMessageMapper.cs#L15-L18

Reply:
Hrm, I might leave them as a separate change